src/assets/css/ProfilePage.css

```diff
@@ -0,0 +1,64 @@
+.container {
+    display: flex;
+}
+
+.some-page-wrapper {
+    display: flex;  
+    flex-direction: column;
+    width: 100%;
+    
+}
+
+.column{
+    display: flex;
+    height: 607px;
+    width: 784px;  
+    align-self: center;
+    border: 1px solid white;  
+    border-radius: 25px;
+    flex-wrap: wrap;
+    
+    
+}
+
+.Save-button {
+    margin-left: 41%;
```

Comment:
This feels pretty dirty; usually we want to keep numbers like these in a way that can be expressed with rational numbers (like 25% = 1/4, 33% = 1/3, etc.). There's probably a much better way to center this.